coordinated-entry-screening/static/css/custom.css

```diff
@@ -14,12 +14,21 @@ html {
 body {
     font-family: 'Libre Franklin', sans-serif;
     margin-bottom: 60px;
+    background-color: #f7f7f7;
 }
 
 h1, h2, h3 {
     text-transform: uppercase;
 }
 
+a {
+    color: var(--blue);
```

Comment:
this is a cool css trick! I haven't seen it before. Looks like it works on all modern browsers https://www.w3schools.com/cssref/sel_root.asp